hepcrawl/utils.py

```diff
@@ -194,3 +194,30 @@ def get_node(text, namespaces=None):
         for ns in namespaces:
             node.register_namespace(ns[0], ns[1])
     return node
+
+
+def coll_cleanforthe(coll):
+    """ Cleanup collaboration, try to find author """
+    author = None
+    re_for_the = re.compile(r'\b(?:for|on behalf of|representing)\b', re.IGNORECASE)
```

Comment:
Ỳou can move this at the beginning of the file at the global level, so that it's not compiled everytime `coll_cleanforthe(coll)` is called.